src/assets/database.sql

```diff
@@ -341,6 +341,26 @@ INSERT INTO `commands` (`name`, `description`, `examples`, `tool`, `type`, `comm
 ('Time sec', 'Seconds to delay the DBMS response', 'sqlmap --time-sec 5', 26, 'input', '--time-sec', '5'),
 ('Techniques', 'SQL injection techniques to use', 'sqlmap --technique BEUSTQ', 26, 'input', '--technique', 'BEUSTQ');
 
+INSERT INTO `commands` (`name`, `description`, `examples`, `tool`, `type`, `command`, `example`, `sudo`, `category`) VALUES 
+('Set --enum switch', 'Shortcut option equivalent to --threads 5 -s 15 -w.',  NULL, 14, 'checkbox', '--enum', NULL, NULL, 'GENERAL OPTIONS'),
+('Skip reverse lookup', 'Skip the reverse lookup (reverse DNS searc) operations.', NULL, 14, 'checkbox', '--noreverse', NULL, NULL, 'GENERAL OPTIONS'),
+('No colors','Disable ANSIColor output, i.e., disables output with colored text.', NULL, 14, 'checkbox', '--nocolor', NULL, NULL, 'GENERAL OPTIONS'),
+('Private ips at the end', 'Show and save private ips at the end of the file domain_ips.txt.', NULL, 14, 'checkbox', '--private', NULL, NULL, 'GENERAL OPTIONS'),
+('File to save subdomains','Write all valid subdomains to the chosen file.', 'dnsenum --subfile subdomains_obtained.txt example.com', 14, 'input', '--subfile', NULL, NULL, 'GENERAL OPTIONS'),
+('Timeout setting','The tcp and udp timeout values in seconds. Defines he interval of time in which dnsenum waits from an answer from the DNS server (default: 10s).', 'dnsenum -t 10 example.com', 14, 'input', '-t', NULL, NULL, 'GENERAL OPTIONS'),
+('Threads setting', 'The number of threads that will perform different queries.', 'dnsenum --threads 5 example.com', 14, 'checkbox', '--threads', NULL, NULL, 'GENERAL OPTIONS'),
+('Verbose output', 'Be verbose: show all the progress and all the error messages.', NULL, 14, 'checkbox', '-v', NULL, NULL, 'GENERAL OPTIONS'),
+('Google pages scrapping', 'The number of google search pages to process when scraping names, the default is 5 pages, the -s switch must be specified.', 'dnsenum -p 15 - 20 example.com', 14, 'input', '-p', NULL, NULL, 'GOOGLE SCRAPPING OPTIONS'),
+('Subdomains scrapping', 'The maximum number of subdomains that will be scraped from Google (default 15)', 'dnsenum -s 40 example.com', 14, 'input', '-s', NULL, NULL, 'GOOGLE SCRAPPING OPTIONS: SUBDOMAINS'),
+('Choose the brute force file', 'Read subdomains from this file to perform brute force. (Takes priority over default dns.txt)', 'dnsenum -f subdomains.txt example.com', 14, 'input', '-f', NULL, NULL, 'SUBDOMAIN BRUTE FORCE'),
+('Update brute force file', 'This command updates the file specified with the -f switch with the follwing parameters: a (all): Update using all results; g; Update using only google scraping results; r: Update using only reverse lookup results; z: Update using only zonetransfer results.', 'dnsenum -f subdmains.txt -u a example.com', 14, 'input', '-u', NULL, NULL, 'BRUTE FORCE OPTIONS: UPDATE'),
```

Comment:
I suppose this `--update` option would be better implemented as `select` command type since there are a few specific values allowed.